test/model/src/include/model/kv_table_item.h

```diff
@@ -70,25 +70,38 @@ class kv_table_item {
      */
     bool contains_any(const data_value &value, timestamp_t timestamp = k_timestamp_latest);
 
+    /*
+     * kv_table_item::exists --
+     *     Check whether the latest value exists.
+     */
+    bool exists();
```

Comment:
`const`? Here and a bunch of other places in this class.

Reply:
I wish this would be possible, but these functions cannot be made `const` because they take lock (or call functions that take lock), so they actually need read-write access to the instance's fields.

Reply:
Ah, that's annoying. I guess if you were really keen you could mark the mutex as `mutable` and then get away with these functions being `const` - but that's optional and there's no need to block the PR on it.

Reply:
I didn't know about this keyword! Let me use it—hopefully it helps.